docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+
+This section is designed to provide you with all the necessary information you 
+need to get started with Inductiva API. Here, we've condensed everything so you 
+can install Inductiva, grab usage tips, and find some of its essential features, 
+with ease.
+
+It's all you need to get familiar and make the most out of Inductiva from the 
+get-go.
+
+In this section you will find:
+
+* [Overview]()
+    * [What is Inductiva?]()
+    * [Who is it for?]()
+    * [What can you do?]()
+    * [Why choose Inductiva?]()
+* [Prerequisites]()
+* [Installing Inductiva]()
+* [What to read next]()
+* [Troubleshooting]()
+* [Frequently Asked Questions (FAQs)]()
+* [Useful Terminology]()
+
+
+## Overview
+### What is Inductiva?
+Inductiva is a state-of-the-art Python package that simplifies running large-scale 
+physical system simulations. Just two lines of code unleash its potentional, freeing 
+you from local hardware limits and the technical hassles of scaling simulations. 
+It's all about making big computational tasks a breeze.
+
+### Who is it for?
+Inductiva is perfect for curious minds in research, science, and engineering who 
+are engaged in intensive computational work. Whether you're unraveling the mysteries 
+of molecular dynamics, exploring the complexities of fluid dynamics, or solving 
+complex engineering problems, Inductiva provides the computational power and simplicity 
+you need. It's perfect for those who want to push the boundaries of simulation-based 
+research without getting bogged down in the technicalities of simulation setup and 
+execution. It turns brain-busting calculations into child's play, so you can spend 
+more time having those 'eureka!' moments and less time waiting for your simulations 
+to run.
+
+### What can you do with Inductiva API?
+With Inductiva, you have the flexibility to scale your simulations according to 
+your needs:
+
+* For a quick demo, there are [ready-to-use simulators]() that allow you to start 
+simulations immediately with your existing configuration files. No fuss over 
+installations or tangled software wires. 
+
+* For those who dream big and simulate bigger, there's the option to [build your own]() 
+digital dream team of machines, known as a ['MachineGroup'](). This feature lets 
+you configure a custom-tailored squad of virtual machines ready to tackle those 
+heavyweight simulations. 
+
+So, whether you're in for a quick demo dash or for comprehensive, detailed analyses, 
+Inductiva is equipped to facilitate your work seamlessly.
+
+### Why choose Inductiva API?
+Because choosing Inductiva means opting for simplicity and efficiency in your 
+simulation work. With just two lines of code, you can activate its powerful features. 
+No need to tangle with technical headaches or a web of dependencies. Inductiva 
+streamlines the process, so can you cut through the clutter and focus on what 
+really matters – *the science*.
+
+## Prerequisites
+
+### Python Installation
```

Comment:
I would not add this to the main page.
Our users will be python users almost for sure, or will have familiarity with it.
I think this could be part of a "Trouble Shooting" section, but I would not use such valuable real estate for this.